### PR TITLE
Topic/auto text update

### PR DESCRIPTION
**Pull request by JAS-lzn**
Updates the text that reminds users to only use `m blueprints` to generate workflow files. Should be clearer now

### CODE REVIEW

File: packages/python/m/ci/docker/gh_workflow_multi.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 from .gh_workflow_single import Workflow as DefaultWorkflow
 
 TEMPLATE = """\
-# AUTOGENERATED FILE - Update {m_dir}/m.yaml
-#   then run `m blueprints --update-workflow` to update
+# AUTOGENERATED FILE - Do not update by hand!
+# Edit  {m_dir}/m.yaml then run
```
Comment (jmlopez-rod) [inline packages/python/m/ci/docker/gh_workflow_multi.py:6]: Only one space between `Edit` and `{m_dir}/m.yaml` please

Comment (JAS-lzn) [reply packages/python/m/ci/docker/gh_workflow_multi.py]: > Only one space between `Edit` and `{m_dir}/m.yaml` please

doneski

**Review by jmlopez-rod — APPROVED** [review]

**Comment by JAS-lzn** [comment]
@jmlopez-rod 

I removed the references to the newly-dead CLI option in the autogenerated text :crab:

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ The format of this changelog is based on
 
 - Wraps colors around `\[` and `\]` for colors in `$PS1`. See
   https://unix.stackexchange.com/a/157884
+- Update text for autogenerated files
```
Comment (jmlopez-rod) [inline CHANGELOG.md:16]: should be under the unreleased section.

Comment (JAS-lzn) [reply CHANGELOG.md]: Learning-to-read-scaled

**Review by jmlopez-rod — APPROVED** [review]

**Pull request merged**